Mundipagg/Models/Request/ListInvoicesRequest.cs

```diff
@@ -7,12 +7,18 @@ namespace Mundipagg.Models.Request
     [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
     public class ListInvoicesRequest : BaseListRequest
     {
+        public string Id { get; set; }
+
         public string Code { get; set; }
 
         public DateTime? CreatedSince { get; set; }
 
         public DateTime? CreatedUntil { get; set; }
 
+        public DateTime? DueSince { get; set; }
+
+        public DateTime? Dueuntil { get; set; }
```

Comment:
I believe that it must be "DueUntil", right?

Reply:
Yes, I will go up correcting this property.